flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/Benchmark.java

```diff
@@ -71,37 +70,57 @@ public static void printHelp() {
         System.out.println();
     }
 
-    @SuppressWarnings("unchecked")
     public static void executeBenchmarks(CommandLine commandLine) throws Exception {
         String configFile = commandLine.getArgs()[0];
-        Map<String, ?> benchmarks = BenchmarkUtils.parseJsonFile(configFile);
-        System.out.println("Found benchmarks " + benchmarks.keySet());
+        Map<String, Map<String, Map<String, ?>>> benchmarks =
+                BenchmarkUtils.parseJsonFile(configFile);
+        System.out.println("Found " + benchmarks.keySet().size() + " benchmarks.");
+        String saveFile = commandLine.getOptionValue(OUTPUT_FILE_OPTION.getLongOpt());
 
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         StreamTableEnvironment tEnv = StreamTableEnvironment.create(env);
 
-        List<BenchmarkResult> results = new ArrayList<>();
-        for (Map.Entry<String, ?> benchmark : benchmarks.entrySet()) {
-            LOG.info("Running benchmark " + benchmark.getKey() + ".");
-
-            BenchmarkResult result =
-                    BenchmarkUtils.runBenchmark(
-                            tEnv, benchmark.getKey(), (Map<String, ?>) benchmark.getValue());
-
-            results.add(result);
-            LOG.info(BenchmarkUtils.getResultsMapAsJson(result));
+        Map<String, Map<String, Map<String, ?>>> results = new HashMap<>();
+        String benchmarkResultsJson = "{}";
+        int index = 0;
+        for (String benchmarkName : benchmarks.keySet()) {
```

Comment:
Since we need to get both the key and the value of this entry, would it be simpler to use `for (Map.Entry<String, Map<String, Map<String, ?>>> entry : benchmarks.entrySet())` here? 

Then we can use `entry.getKey()` and `entry.getValue()`, instead of repeatedly calling `results.get(benchmarkName)` and `benchmarks.get(benchmarkName)`, in the loop body.